test/onnx_import.cpp

```diff
@@ -195,6 +195,77 @@ TEST(onnx, model_conv2d_strides_assymetric_padding)
     EXPECT_EQ(expected_output, result.front());
 }
 
+TEST(onnx, model_average_pool_2d)
+{
+    // Pooling with strides=2 and no padding
+    auto model{ngraph::onnx_import::import_onnx_function(
+        ngraph::file_util::path_join(SERIALIZED_ZOO, "onnx/average_pool_2d.onnx"))};
+
+    // input data shape (1, 1, 4, 4)
+    Inputs inputs;
+    inputs.push_back(test::NDArray<float, 4>({{{{0.f, 1.f, 2.f, 3.f},
+                                                {4.f, 5.f, 6.f, 7.f},
+                                                {8.f, 9.f, 10.f, 11.f},
+                                                {12.f, 13.f, 14.f, 15.f}}}})
+                         .get_vector());
+
+    // (1, 1, 2, 2)
+    auto expected_output =
+        ngraph::test::NDArray<float, 4>({{{{2.5f, 4.5f}, {10.5f, 12.5f}}}}).get_vector();
```

Comment:
remove `ngraph::` - no need for it, at the top of this file see `using namespace ngraph;`.